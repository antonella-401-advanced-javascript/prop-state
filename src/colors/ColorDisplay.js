import React from 'react';
import PropTypes from 'prop-types';

const ColorDisplay = ({ backgroundColor }) => (
  <div style={{
    backgroundColor,
    width: '300px',
    height: '300px'
  }}></div>
);

ColorDisplay.propTypes = {
  backgroundColor: PropTypes.string.isRequired
};

export default ColorDisplay;