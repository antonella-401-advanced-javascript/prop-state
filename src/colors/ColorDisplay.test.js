import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

Enzyme.configure({ adapter: new Adapter() });
describe('ColorDisplay component', () => {
  it('renders ColorDisplay', () => {
    const wrapper = shallow(<ColorDisplay backgroundColor='red' />);
    expect(wrapper).toMatchSnapshot();
  });
});