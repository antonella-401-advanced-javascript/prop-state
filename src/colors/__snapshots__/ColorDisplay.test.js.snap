// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component renders ColorDisplay 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "height": "300px",
      "width": "300px",
    }
  }
/>
`;
